pandas/_libs/tslib.pyx

```diff
@@ -757,7 +758,8 @@ cdef inline ignore_errors_out_of_bounds_fallback(ndarray[object] values):
 
 @cython.wraparound(False)
 @cython.boundscheck(False)
-cdef array_to_datetime_object(ndarray[object] values, bint is_raise,
+cdef array_to_datetime_object(ndarray[object] values,
```

Comment:
really? we already pass thru errors so why do u need yet another arg?

Reply:
Agreed. Instead of passing `is_raise`& `is_coerce` just pass down `errors` from `array_to_datetime`.  Please update the docstring of `array_to_datetime_object` after that change.

Reply:
What exactly is the objection to using `is_raise` and `is_coerce`?

We use them all over `array_to_datetime`.  By passing through `errors`, you're essentially duplicating the logic `array_to_datetime` does to determine these flags, which you ultimately use in `array_to_datetime_object`.

Reply:
That's fair. Since this is essentially a private method too changing the signature shouldn't be that big of a deal

Reply:
no, we already consistency pass around errors=, I think better here rather than having 2 args that are passed that could actually be in conflict.